splinterd/src/daemon.rs

```diff
@@ -646,6 +671,11 @@ impl SplinterDaemonBuilder {
         self
     }
 
+    pub fn with_db_url(mut self, value: String) -> Self {
```

Comment:
This should be guarded by the "database" feature (as well as the struct's field).

Reply:
updated